pallets/allocations/Cargo.toml

```diff
@@ -16,15 +16,18 @@ std = [
   "sp-io/std",
   "sp-runtime/std",
   "sp-std/std",
+  "pallet-membership/std",
```

Comment:
We shouldn't need a dependency to `pallet-membership`, for the tests and as dev dependency it would be totally okay.